modules/con_quay_single_prereq.adoc

```diff
@@ -2,28 +2,23 @@
 
 ifeval::["{productname}" == "Red Hat Quay"]
 //* **Red Hat Enterprise Linux (RHEL)**: Obtain the latest Red Hat Enterprise Linux 7 server media from the link:https://access.redhat.com/downloads/content/69/ver=/rhel---7/7.9/x86_64/product-software[Downloads page] and follow the installation instructions from the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/index[Red Hat Enterprise Linux 7 Installation Guide].
-* **Red Hat Enterprise Linux (RHEL)**: Obtain the latest Red Hat Enterprise Linux 8 server media from the link:https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.3/x86_64/product-software[Downloads page] and follow the installation instructions available in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/[Product Documentation for Red Hat Enterprise Linux 8].
+* **Red Hat Enterprise Linux (RHEL)** 8: Obtain the latest Red Hat Enterprise Linux 8 server media from the link:https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.3/x86_64/product-software[Downloads page] and follow the installation instructions available in the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/[Product Documentation for Red Hat Enterprise Linux 8].
 * **Valid Red Hat Subscription**: Configure a valid Red Hat Enterprise Linux 8 server subscription.
 endif::[]
 * **CPUs**: Two or more virtual CPUs.
 * **RAM**: 4GB or more.
 * **Disk space**:  The required disk space depends on the storage needs for the registry. Approximately 30GB of disk space should be enough for a test system, broken down as follows:
 ** At least 10GB of disk space for the Red Hat Enterprise Linux operating system.
 ** At least 10GB of disk space for docker storage (to run 3 containers).
-** At least 10GB of disk space for Quay local storage. Note that CEPH or other local storage might require more memory. 
+** At least 10GB of disk space for Quay local storage. Note that CEPH or other local storage might require more memory.
 
 More information on sizing can be found at link:https://access.redhat.com/articles/5177961[Quay 3.x Sizing Guidlines].
 
-[NOTE]
-====
```

Comment:
as above, I'd leave this is if rhel 7 hasn't been deprecated - just change the version numbers